android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/DDLFormView.java

```diff
@@ -50,7 +55,7 @@
 		DEFAULT_LAYOUT_IDS.put(Field.EditorType.DECIMAL, R.layout.ddlfield_number_default);
 		DEFAULT_LAYOUT_IDS.put(Field.EditorType.RADIO, R.layout.ddlfield_radio_default);
 		DEFAULT_LAYOUT_IDS.put(Field.EditorType.SELECT, R.layout.ddlfield_select_default);
-		DEFAULT_LAYOUT_IDS.put(Field.EditorType.TEXT, R.layout.ddlfield_text_default);
+		DEFAULT_LAYOUT_IDS.put(TEXT, R.layout.ddlfield_text_default);
```

Comment:
why?

Reply:
Because Android Studio changed it to static import and I was lazy :P